spark/src/test/scala/org/apache/spark/sql/delta/GeneratedColumnSuite.scala

```diff
@@ -622,7 +622,7 @@ trait GeneratedColumnSuiteBase
       s"${generatedColumnType.sql} NOT NULL"
     }
     test(s"validateGeneratedColumns: column type ${columnTypeString}" +
-        s" $verb expression type ${generateAsExpression.expr.sql}") {
+        s" $verb expression type $generateAsExpression") {
```

Comment:
This one is a bit silly. We cannot use `expr` yet, because that relies on a `SparkSession` which is only initialised after the test is defined. We side step this issue by using `Column.toString()` which uses something similar to `.expr.sql`.